### PR TITLE
Feature/audits

### PR DESCRIPTION
**Pull request by skuhlmann**
https://app.clickup.com/t/1eqpczb

first version:
- new route /settings/audit
- link from the setting page
- table with time, member address and json of the update
- used an accordion to approximate the design

### CODE REVIEW

File: src/components/auditJsonDisplay.jsx [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+          <AccordionButton>
+            <AccordionIcon />
+            <Box flex='1' textAlign='left' ml={1}>
+              {updateName(log)}
+            </Box>
+          </AccordionButton>
+        </h2>
+        <AccordionPanel pb={4}>
+          <pre
+            /* eslint-disable */
```
Comment (jordanlesich) [inline src/components/auditJsonDisplay.jsx]: Curious why we're setting inner HTML here. Is this part to visualise JSON?

File: src/pages/MetaAudit.jsx [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+              <Tr>
+                <Th fontSize='md'>Created At</Th>
+                <Th fontSize='md'>Updated By</Th>
+                <Th fontSize='md' w='70%'>
+                  JSON Update
+                </Th>
+              </Tr>
+            </Thead>
+            <Tbody>
+              {daoMetaData?.logs
```
Comment (jordanlesich) [inline src/pages/MetaAudit.jsx]: This could useMemo to sort if possible. Not a huge deal if it's a pain.

Comment (skuhlmann) [reply src/pages/MetaAudit.jsx]: I like this. will update.

File: src/components/auditJsonDisplay.jsx (hunk 1 above)
Comment (skuhlmann) [reply src/components/auditJsonDisplay.jsx]: found way to do this without less dangerously

**Pull request merged**